### PR TITLE
Handle FWMT CREATE from CaseCreated RM message

### PR DESCRIPTION
**Pull request by NickGrantONS**
# Motivation and Context
We want to drive FWMT via regular RM events. Case Processor was re-using the mechanism that Action Worker uses to send messages directly to Fieldwork Adapter, so we wanted to refactor that away and only use the regular CaseCreated RM event.

# What has changed
Got rid of any and every reference to the old direct message. Ensured that a CaseCreated RM event is emitted when a CCS case is created.

# How to test?
Run the ATs. There should be zero regression.

# Links
Trello: https://trello.com/c/ONAETtD2

### CODE REVIEW

**Comment by NickGrantONS** [comment]
[Refactor CCSPropertyListed features (5)](https://trello.com/c/ONAETtD2/512-refactor-ccspropertylisted-features-5)

File: src/main/java/uk/gov/ons/census/fwmtadapter/messaging/CaseUpdatedReceiver.java [hunk 1]
```diff
@@ -53,4 +96,15 @@ private boolean canIgnoreEvent(ResponseManagementEvent event) {
     return event.getPayload().getMetadata() == null
         || event.getPayload().getMetadata().getFieldDecision() == null;
   }
+
+  private Boolean getCEComplete(CollectionCase caze) {
+    if (caze.getAddress().getAddressType().equals("CE")
+        && caze.getAddress().getAddressLevel().equals("E")) {
+      if (caze.getReceiptReceived() != null) {
```
Comment (AdamHawtin) [inline src/main/java/uk/gov/ons/census/fwmtadapter/messaging/CaseUpdatedReceiver.java]: Could be anded instead of another nested `if`?

File: src/main/java/uk/gov/ons/census/fwmtadapter/model/dto/fwmt/FwmtCreateActionInstruction.java [hunk 2]
```diff
@@ -27,5 +27,5 @@
   private boolean handDeliver;
   private Integer ceExpectedCapacity;
   private Integer ceActualResponses;
-  private boolean undeliveredAsAddress; // TODO: This will probably be removed
+  private Boolean undeliveredAsAddress; // TODO: This will probably be removed
```
Comment (AdamHawtin) [inline src/main/java/uk/gov/ons/census/fwmtadapter/model/dto/fwmt/FwmtCreateActionInstruction.java]: Why this change? We default UAA to false on sample load anyway.

Comment (NickGrantONS) [reply src/main/java/uk/gov/ons/census/fwmtadapter/model/dto/fwmt/FwmtCreateActionInstruction.java]: it's `Boolean` in the `CollectionCase` source, so it's `Boolean` here in case of garbage in, garbage out. Have removed the comment (which was there before)

File: src/main/java/uk/gov/ons/census/fwmtadapter/messaging/CaseUpdatedReceiver.java (hunk 1 above)
Comment (NickGrantONS) [reply src/main/java/uk/gov/ons/census/fwmtadapter/messaging/CaseUpdatedReceiver.java]: done
